bin/node-sass

```diff
@@ -309,15 +316,15 @@ function run(options, emitter) {
     emitter.emit('error', 'The --source-map option must be either a boolean or directory when compiling a directory');
   }
 
-  if (options.importer) {
+  if (options.importer && typeof options.importer === 'string') {
```

Comment:
I am not completely sure about this check. I had first used `util.isString()`, but that is deprecated. I also did not want to add additional dependencies like `lodash.isstring`.